cmd/apiserver-boot/boot/run/local.go

```diff
@@ -71,7 +73,8 @@ func AddLocal(cmd *cobra.Command) {
 	localCmd.Flags().StringVar(&server, "apiserver", "", "path to apiserver binary to run")
 	localCmd.Flags().StringVar(&controllermanager, "controller-manager", "", "path to controller-manager binary to run")
 	localCmd.Flags().StringVar(&etcd, "etcd", "", "if non-empty, use this etcd instead of starting a new one")
-
+	localCmd.Flags().StringSliceVar(&ControllerArgs, "controller-args", []string{}, "args passed to controller-manager")
+	localCmd.Flags().StringSliceVar(&ApiserverArgs, "apiserver-args", []string{}, "args passed to apiserver")
```

Comment:
@magicsong thx for developing this. might have a question before further review:

instead of add new flags here, can we add existing flags by those `AddFlagSets` caller to append server/controller's flags dynamically? initially it would be like:

```suggestion
	switch {
	        case "apiserver":
	                apiserverRecommendedOptions.AddFlags(...)
	        case "controller-manager":
	                controllerManagerOptions.AddFlags(...)
	}
	....
	exec.Command(server, flags.Args()...)
```
wdyt?

Reply:
I think it is not good because it will confuse users. These flags are copied from `apiserver-boot run in-cluster` mode，the usage of `in-cluster` and `local` should be similar. In addition, i can't find a suitable existing flag to 'hold' this function.

Reply:
> I think it is not good because it will confuse users. 

by managing flags into logical sections, it would be fine. or we could name a new section for the command itself. 

ref: [https://github.com/kubernetes/kubernetes/pull/64517](https://github.com/kubernetes/kubernetes/pull/64517)

Reply:
Looks good. But we would change a lot of codes to make the styles of all sub command uniform. I suggest making a plan for it and it can be low priority. What do you think?